lib/inets/src/http_server/httpd_request_handler.erl

```diff
@@ -149,7 +149,7 @@ continue_init(Manager, ConfigDB, SocketType, Socket, Peername, Sockname,
     case Result of
         error ->
             httpd_util:error_log(ConfigDB,
-                                 httpd_logger:error_report('TLS', Status,
+                                 httpd_logger:error_report(internal, Status,
```

Comment:
I do not think we should make it internal, just not hardcode TLS. You should use SocketType to determine if it should be TLS or TCP. See httd_logger.erl